src/ontology/cl-edit.owl

```diff
@@ -31956,6 +31965,47 @@ AnnotationAssertion(rdfs:label obo:CL_4042002 "TAC3-positive medium spiny neuron
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37232741") oboInOwl:hasRelatedSynonym obo:CL_4042003 "CNS-associated macrophage")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29426702") Annotation(oboInOwl:hasDbXref "PMID:37232741") rdfs:comment obo:CL_4042003 "A border associate macrophage expresses CD45 and CD11B. This cell is also known to have the following molecular signature: CD206, CD38, LYVE1, CD163, CD169.")
+AnnotationAssertion(rdfs:label obo:CL_4042003 "border associated macrophage"@en)
+SubClassOf(obo:CL_4042003 obo:CL_0000878)
+
+# Class: obo:CL_4042004 (choroid epiplexus macrophage)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31061494") Annotation(oboInOwl:hasDbXref "PMID:37232741") obo:IAO_0000115 obo:CL_4042004 "A choroid plexus macrophage that is part of the apical surface of some choroid plexus epithelium. This macrophage has a star-like shaped body.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042004 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042004 "2024-03-26T19:14:58Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31433571") oboInOwl:hasExactSynonym obo:CL_4042004 "Kolmer’s cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:38347231") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042004 "cpepiMΦ")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:38347231") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042004 "epiplexus cpMΦ")
+AnnotationAssertion(rdfs:label obo:CL_4042004 "choroid epiplexus macrophage"@en)
+SubClassOf(obo:CL_4042004 obo:CL_0000880)
+
+# Class: obo:CL_4042005 (stromal choroid plexus macrophage)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:38347231") obo:IAO_0000115 obo:CL_4042005 "A choroid plexus macrophage that is part of some choroid plexus stroma.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042005 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042005 "2024-03-26T19:21:51Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:38347231") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042005 "stromal cpMΦ")
+AnnotationAssertion(rdfs:label obo:CL_4042005 "stromal choroid plexus macrophage"@en)
+SubClassOf(obo:CL_4042005 obo:CL_0000880)
+
+# Class: obo:CL_4042006 (dural macrophage)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31061494") obo:IAO_0000115 obo:CL_4042006 "A central nervous system macrophage which is part of some dura matter. This macrophage phagocytoses intruding pathogens and foreign molecules detected in the bloodstream or in the cerebrospinal fluid. This cell has an amoeboid body with dynamic protrusions in homeostasis.")
```

Comment:
Consider using the parent term to define it: border associated macrophage.
Consider substituting "some" for "a".